src/transport/BusCapacityByNumberOfSeats.java

```diff
@@ -0,0 +1,45 @@
+package transport;
+
+public enum BusCapacityByNumberOfSeats {
+    EXTRA_SMALL(0, 10),
+    SMALL(10, 25),
+    MEDIUM(40, 60),
+    LARGE(60, 80),
+    EXTRA_LARGE(100, 120);
+
+    private Integer from;
+    private Integer to;
```

Comment:
```suggestion
    private int from;
    private int to;
```
раз null-ми быть не могут